browser/brave_browser_main_parts.cc

```diff
@@ -99,11 +100,13 @@ void BraveBrowserMainParts::PreBrowserStart() {
 #endif
 
 #if BUILDFLAG(IS_WIN)
-  // As this uses first run sentinel time, PreBrowserStart() is good place to
-  // initialize.
-  day_zero_browser_ui_expt_manager_ =
-      std::make_unique<DayZeroBrowserUIExptManager>(
-          g_browser_process->profile_manager());
+  // As DayZeroBrowserUIExptManager uses first run sentinel time,
+  // PreBrowserStart() is good place to initialize.
+  if (base::FeatureList::IsEnabled(features::kBraveDayZeroExperiment)) {
```

Comment:
I'd prefer to add the static Create method that checks the feature state and first run precondition.

Reply:
Added static Create method and did some more cleanup. thanks!